frontend/components/Button/index.css

```diff
@@ -0,0 +1,15 @@
+@import '../../styles/variables.css';
+@import '../../styles/colors.css';
+@import '../../styles/layout.css';
+
+.root {
+  display: flex;
+  box-sizing: border-box;
+  padding: $spacing-12 $spacing-20;
+
+  background-color: $yellow;
+
+  color: inherit;
```

Comment:
Isn't this default behaviour?

Reply:
The link has that weird things that change the color of the text. Not sure if I really need this one, but it is just to be sure that it resets them